File: C/soma_impares.c
#include <stdio.h>
#include <locale.h>  // Inclusão da biblioteca de localização

int main(){
    setlocale(LC_ALL, "");  // Configura as opções de localização

    int X, Y, soma, troca;  // Declaração de variáveis

    soma = 0;  // Inicialização da variável que armazenará a soma dos números ímpares

    // Leitura dos dois números
    printf("Digite dois numeros:\n");
    scanf("%d", &X);
    scanf("%d", &Y);

    // Verificação da ordem dos números e troca se necessário
    if (X > Y){
        troca = X;  // Armazenamento temporário do valor de X
        X = Y;      // Troca dos valores
        Y = troca;
    }

    // Loop para somar os números ímpares entre X e Y
    for (int i = X+1; i < Y; i++) {
        if (i % 2 != 0) {  // Verificação se o número é ímpar
            soma = soma + i;  // Adição do número ímpar à soma total
        }
    }

    // Exibição do resultado
    printf("SOMA DOS IMPARES = %d\n", soma);

    return 0;
}

File: Java/soma_impares.java
import java.util.Locale;
import java.util.Scanner;

public class soma_impares {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite dois números:");
        int x = sc.nextInt();
        int y = sc.nextInt();

        int soma = 0, troca;

        if (x > y){
            troca = x;
            x = y;
            y = troca;
        }

        for (int i = x+1; i < y; i++){
            if (i % 2 != 0){
                soma += i;
            }
        }

        System.out.println("SOMA DOS IMPARES = " + soma);

        sc.close();

    }
}
